src/tiled/commanddialog.cpp

```diff
@@ -45,6 +45,9 @@ CommandDialog::CommandDialog(QWidget *parent)
 
     setWindowTitle(tr("Edit Commands"));
     Utils::restoreGeometry(this);
+
+    connect(mUi->pushButton, &QPushButton::clicked, 
+            [this]() { CommandManager::instance()->populateMenu(); });
```

Comment:
I realised another bug, if the user clicks on the cross button of the dialog instead of clicking on the close button, the results don't get committed! I'll fix this in the next commit.